demisto_sdk/commands/content_graph/interface/neo4j/import_utils.py

```diff
@@ -60,37 +53,26 @@ def ensure_data_uniqueness(self) -> None:
     def _get_import_sources(self) -> Set[str]:
         sources: Set[str] = set()
         for filename in self.import_path.iterdir():
-            # csv filenames are in the format <source>.<nodes/relationships>.<labels/type>.csv
-            if source := filename.name.split(".")[0]:
-                sources.add(source)
+            if "graphml" in filename.suffix:
```

Comment:
```suggestion
           filename.suffox == ".graphml":
```
And maybe move it to constant?